services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -101,20 +100,6 @@ public void testRouteOptionsFromUrl() throws IOException {
     );
   }
 
-  @Test
-  public void testRouteOptionsFromUrl_alreadyDecoded() throws IOException {
```

Comment:
This test was wrong - if a URL is decoded in full, it means that it's not a valid URL anymore (it contains unescaped reserved characters), and it's expected that `RouteOptions.fromUrl` might fail.